presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.sqlserver;
+
+import io.prestosql.tests.AbstractTestIntegrationSmokeTest;
+import org.testng.annotations.Test;
+
+import static io.airlift.tpch.TpchTable.ORDERS;
+import static io.prestosql.plugin.sqlserver.SqlServerQueryRunner.createSqlServerQueryRunner;
+import static io.prestosql.plugin.sqlserver.TestingSqlServerServer.executeInSqlServer;
+import static java.lang.String.format;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+@Test
+public class TestSqlServerIntegrationSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    public TestSqlServerIntegrationSmokeTest()
+    {
+        super(() -> createSqlServerQueryRunner(ORDERS));
+    }
+
+    @Test
+    public void testDropTable()
```

Comment:
This looks like it should belong to `AbstractTestIntegrationSmokeTest`

Reply:
Let me fix this in separate PR. Add this `AbstractTestIntegrationSmokeTest` could blow up.

Reply:
Fine, so don't add them here.

Reply:
hm.. then add `// TODO move this test to AbstractTestIntegrationSmokeTest`

Reply:
This TODO is incorrect. The smoke tests intentionally only have read queries, so that they apply to all connectors. Maybe we should have an extended version of it that is read/write. Though we already have them in distributed test queries...